Content.Server/Fugitive/FugitiveSystem.cs

```diff
@@ -0,0 +1,126 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Content.Server.Fax;
+using Content.Server.GameTicking.Rules.Components;
+using Content.Server.GameTicking;
+using Content.Server.Station.Components;
+using Content.Server.Station.Systems;
+using Content.Server.Fugitive;
+using Content.Server.Mind.Components;
+using Content.Server.NPC.Systems;
+using Content.Server.Mind;
+using Content.Server.Objectives;
+using Content.Server.Objectives.Interfaces;
+using Content.Server.Roles;
+using Content.Shared.Roles;
+using Content.Shared.CCVar;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Configuration;
+using Robust.Shared.Random;
+using Timer = Robust.Shared.Timing.Timer;
+
+namespace Content.Server.Fugitive;
+
+public sealed class FugitiveSystem: EntitySystem
+{
+    [Dependency] private readonly StationSystem _stationSystem = default!;
+    [Dependency] private readonly FaxSystem _faxSystem = default!;
+    [Dependency] private readonly FactionSystem _faction = default!;
+    [Dependency] private readonly MindSystem _mindSystem = default!;
+    [Dependency] private readonly IObjectivesManager _objectivesManager = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly IConfigurationManager _configManager = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] protected readonly GameTicker GameTicker = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<FugitiveComponent, MindAddedMessage>(OnMindAdded);
+    }
+
+    /// <summary>
+    ///   When a mind is added to a fugitive, send wanted notices to all fax machines on all stations.
+    /// </summary>
+    private async void OnMindAdded(EntityUid entity, FugitiveComponent component, MindAddedMessage args)
+    {
+        AddFugitiveRole(entity);
+
+        if (component.WantedNoticeSent)
+            return;
+
+        component.WantedNoticeSent = true;
+
+        // Give the fugitive a head start before sending wanted notices.
+        var minimumWantedNoticeDelay = _configManager.GetCVar(CCVars.SalvageExpeditionFugitiveWantedNoticeMinimumDelay);
+        var maximumWantedNoticeDelay = _configManager.GetCVar(CCVars.SalvageExpeditionFugitiveWantedNoticeMaximumDelay);
+        var delay = _random.Next(minimumWantedNoticeDelay, maximumWantedNoticeDelay);
+
+        // 15 minutes is the typical expedition mission time, so we'll use that as the default and add a random delay.
+        await Task.WhenAll(Timer.Delay(1000 * 60 * (15 + delay)));
+
+        var stations = _stationSystem.GetStations();
+        foreach (var station in stations)
+        {
+            SendWantedNotices(station, entity);
+        }
+    }
```

Comment:
Don't use async like this because it won't work for several reasons.
1. Doesn't work if server restarts.
2. Doesn't work if game mode changes.
3. Doesn't work if game is paused.

Don't use cvars for stuff that should be on the gamerule.

Haven't looked at the rest yet.